src/components/Coaching/CoachingDetail/ActivitySummary/ActivitySummary.tsx

```diff
@@ -175,79 +181,152 @@ export const ActivitySummary: React.FC<ActivitySummaryProps> = ({
                   </TableCell>
                 </DividerRow>
                 <HeaderRow role="rowheader">
-                  <AlignedTableCell colSpan={periods.length + 2}>
-                    {t('Electronic Messages')}
-                  </AlignedTableCell>
-                </HeaderRow>
-                <TableRow>
-                  <AlignedTableCell>{t('Sent')}</AlignedTableCell>
-                  {periods.map(({ startDate, electronicMessageSent }) => (
-                    <AlignedTableCell key={startDate}>
-                      {electronicMessageSent}
+                  <AlignedTableCell>{t('Initiations')}</AlignedTableCell>
+                  {periods.map((period) => (
+                    <AlignedTableCell key={period?.startDate}>
+                      {period?.initiationTotal}
                     </AlignedTableCell>
                   ))}
                   <AlignedTableCell>
-                    {Math.round(averages.electronicMessageSent)}
-                  </AlignedTableCell>
-                </TableRow>
-                <TableRow>
-                  <AlignedTableCell>
-                    {t('Resulting Appointments')}
+                    {Math.round(averages?.initiationTotal)}
                   </AlignedTableCell>
-                  {periods.map(
-                    ({ startDate, electronicMessageWithAppointmentNext }) => (
-                      <AlignedTableCell key={startDate}>
-                        {electronicMessageWithAppointmentNext}
-                      </AlignedTableCell>
-                    ),
-                  )}
-                  <AlignedTableCell>
-                    {Math.round(averages.electronicMessageWithAppointmentNext)}
-                  </AlignedTableCell>
-                </TableRow>
+                </HeaderRow>
+                {activitiesByPhase
+                  .get(PhaseEnum.Initiation)
+                  ?.map((activity) => {
+                    const activityVariableName = snakeToCamel(
+                      toLower(activity),
+                    );
+                    return !isNaN(averages[activityVariableName]) &&
```

Comment:
I believe that Number.isNan is preferred over isNan because the former is more robust. [Reference 1](https://stackoverflow.com/a/33164924) [Reference 2](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN) 

```suggestion
                    return !Number.isNaN(averages[activityVariableName]) &&
```

Reply:
@canac So `Number.isNaN` returns `false` for `undefined` which breaks what we are trying to do. We want to make sure the value given is actually a number. I think I should use `Number.isInteger()` instead. What do you think?

Reply:
Actually nevermind. The values are floats. It seems like `typeof averages[activityVariableName] === 'number'` works well. Thoughts?

Reply:
I think in this case it's better to use `!NaN()`, as `Number.isNaN()` checks to see if the value is `NaN` rather than check if the value is not a number. What are your thoughts?

```
Number.isNaN({});
// <- false, {} is not NaN
Number.isNaN('ponyfoo')
// <- false, 'ponyfoo' is not NaN
Number.isNaN(NaN)
// <- true, NaN is NaN
Number.isNaN('pony'/'foo')
// <- true, 'pony'/'foo' is NaN, NaN is NaN

isNaN({});
// <- true, {} is not a number
isNaN('ponyfoo')
// <- true, 'ponyfoo' is not a number
isNaN(NaN)
// <- true, NaN is not a number
isNaN('pony'/'foo')
// <- true, 'pony'/'foo' is NaN, NaN is not a number
```

Reply:
I think you're right. The `reduce` callback will never execute if periods is empty, so we'll never be dividing by 0, and the average will never be NaN. So `averages[activityVariableName]` will either be a number or undefined. `typeof averages[activityVariableName] === 'number'` seems like a great check! (You can remove the `!== null` check too).

Reply:
Undo what I posted. Lets go with `typeof averages[activityVariableName] === 'number'`

Reply:
Ok sounds good!